src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
```

Comment:
replace 100 with constant
```suggestion
        int randomNumber = random.nextInt(100);
```